libcloudforensics/aws.py

```diff
@@ -30,7 +31,10 @@
 
 EC2_SERVICE = 'ec2'
 ACCOUNT_SERVICE = 'sts'
+# Default Amazon Machine Image to use for bootstrapping instances
+UBUNTU_1804_AMI = 'ami-0013b3aa57f8a4331'
```

Comment:
Where does this come from? Is it a universally accessible repo?

Reply:
This is a public AMI ID that is available on AWS marketplace. Should be available to anyone.